sql/variables/system_variables.go

```diff
@@ -848,7 +848,7 @@ var systemVars = map[string]sql.SystemVariable{
 	"gtid_executed": {
 		Name:              "gtid_executed",
 		Scope:             sql.SystemVariableScope_Global,
-		Dynamic:           false,
+		Dynamic:           true,
```

Comment:
I'd add a note here that this deviates from MySQL

Reply:
actually... adding that comment made me feel icky, so I reverted this line to keep `Dynamic` as false and found the `SystemVariables.AssignValues` method to get around the read-only check. 😉 Thanks for poking on this!! It helped me find the right interface to use and keep the sys var not writable by users to match MySQL.